pkgs/development/python-modules/channels_redis/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv
+, fetchPypi
+, buildPythonPackage
+, redis, msgpack, channels, aioredis
+}:
+buildPythonPackage rec {
+
+  pname = "channels_redis";
+  version = "2.4.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "b7ccbcb2fd4e568c08c147891b26db59aa25d88b65af826ce7f70c815cfb91bc";
+  };
+
+  propagatedBuildInputs = [
+    redis
+    aioredis
+    msgpack
+    channels
+  ];
+
```

Comment:
To enable tests:

```
  src = fetchFromGitHub {
    owner = "django";
    repo = "channels_redis";
    rev = version;
    sha256 = "05niaqjv790mnrvca26kbnvb50fgnk2zh0k4np60cn6ilp4nl0kc";
  };

  checkInputs = [ pytest async_generator ];
  checkPhase = ''
    # ensure tests are running against installed package
    mv channels_redis channels_redis.hidden
    # test_double_receive expects a redis server running
    pytest tests/ -k "not test_double_receive"
  '';
```

Reply:
Is it necessary to pull the GitHub version to get tests working? Or can I just add the `checkInputs` and `checkPhase` portions?

Reply:
Another one bites the dust!

@risicle thank you so much for all of the help!